src/app.js

```diff
@@ -25,8 +29,11 @@ app.use(
 );
 app.use(passport.initialize());
 app.use(passport.session());
+app.use('/api/project', projectRoutes);
+app.use('/api/post', postRoutes);
+app.use('/api/comment', commentRoutes);
+app.use('/api/profile', profileRoutes);
```

Comment:
Make all route names plural. As in `projects`, `posts`, `comments` and so on. Because these represent a collection of resources as per REST standards.

You also need `app.use('/api/myprofile', myProfileRoutes);` (This one doesn't have to be plural because my profile is only 1 profile)